TSOClient/tso.simantics/Engine/VMSuitProvider.cs

```diff
@@ -222,7 +222,7 @@ public static object GetSuit(VMStackFrame context, VMSuitScope scope, ushort id)
                             if (context.VM.TS1) return null;
                             var job = avatar.GetPersonData(VMPersonDataVariable.OnlineJobID);
                             if (job < 1 || job > 5) return null;
-                            var level = Math.Max(0, Math.Min(3, (avatar.GetPersonData(VMPersonDataVariable.OnlineJobGrade) - 1) / 4));
+                            var level = Math.Max(0, Math.Min(2, Convert.ToInt32(Math.Round((avatar.GetPersonData(VMPersonDataVariable.OnlineJobGrade) - 1) / 4d))));
```

Comment:
What is this meant to do? I've got a few complaints.

`Convert.ToInt32` shouldn't be used, cast with `(int)`.
`4d` instead of `4.0` for double literals is weird. Also, avoid using doubles for integer math.

If you want the integer division to round normally, add (divisor / 2) to the value before dividing it. In this case, that would change `- 1` to `+ 1`.

Reply:
```suggestion
                            var level = Math.Max(0, Math.Min(2, (avatar.GetPersonData(VMPersonDataVariable.OnlineJobGrade) + 1) / 4));
```
Maybe this is what you want?